test/integration-tests/test-server/config/server-config.ts

```diff
@@ -0,0 +1,25 @@
+import { ApplicationLogProperties } from '@enigmatis/polaris-logs';
+import { injectable } from 'inversify';
+import { PolarisProperties, PolarisServerConfig } from '../../../../src/main';
+import * as polarisPropertiesPath from './properties.json';
+
+@injectable()
+export class TestServerConfig implements PolarisServerConfig {
+    polarisProperties: PolarisProperties;
+    applicationLogProperties: ApplicationLogProperties;
+
+    constructor() {
+        this.polarisProperties = {
+            endpoint: polarisPropertiesPath.endpoint,
+            port: polarisPropertiesPath.port,
+            includeSubscription: polarisPropertiesPath.includeSubscription,
+        };
+        this.applicationLogProperties = {
```

Comment:
have you thought about separating config file into 2 files? `server-properties` and `logger-properties`